pcsx2/DEV9/ATA/ATA_Info.cpp

```diff
@@ -190,8 +193,8 @@ void ATA::CreateHDDinfo(u64 sizeSectors)
 	index = 80 * 2;
 	//Major revision number (1-3-Obsolete, 4-7-ATA4-7 supported)
 	WriteUInt16(identifyData, &index, 0x70); //word 80
-	//Minor revision number
-	WriteUInt16(identifyData, &index, 0); //word 81
+	//Minor revision number, 0x18 - ATA/ATAPI-6 T13 1410D revision 0
+	WriteUInt16(identifyData, &index, 0x18); //word 81
```

Comment:
I was under the impression that you where wanting to report a ATAPI-7 spec HDD info. It seems your intention here it to match what original HDD reports

Reply:
ATAPI-7 is what PS2 console supports, ATAPI-6 is what official sony HDD supports

Reply:
Also, this is minor supported version. Actual supported version is set in word 80